models/stg_reddit_ads__account.sql

```diff
@@ -30,6 +30,9 @@ final as (
         status,
         time_zone_id,
         view_attribution_window
+
+        {{ fivetran_utils.fill_pass_through_columns('reddit_ads__account_passthrough_metrics') }}
+
```

Comment:
In the other ad reporting packages we provide the passthrough functionality for the report staging and final models. Can you add these similar (but with the respective variable name) configs to the other report models.

Reply:
@fivetran-reneeli looks like this note was missed. Would you be able to add these passthrough columns to the report staging models.

Reply:
oops-- added!